packages/swap/src/handlers/openSwapDepositChannel.ts

```diff
@@ -39,7 +39,7 @@ export default async function openSwapDepositChannel(
     env.CHAINFLIP_NETWORK,
   );
 
-  const { destChain, ...rest } = input;
+  const { destChain, ccmMetadata, ...rest } = input;
```

Comment:
we passed the `ccmMetadata` to the `prisma.swapDepositChannel.upsert` call which resulted in an error because  `ccmMetadata` is not a column in the database

Reply:
We don't want to save it in the database?

Reply:
we want in the future, but the table does not have the columns right now. will save them in a follow up, but for now i would like to fix the working example in the docs 🙂